### ISSUE TITLE
Add Submit button to review form

### ISSUE THREAD

**Issue by hdrury1** [libero/prc#79]
We should make clear to users how they can complete the review form by adding a button to 'submit' the information. This should result in the review being appended to the article and should replace the current blue arrow button affordance.

<img width="191" alt="Screenshot 2020-05-18 at 12 06 44" src="https://user-images.githubusercontent.com/17023702/82206400-10da1980-9900-11ea-9e85-5a09f4b8f964.png">

**Comment by chugginselifesciences** [comment]
Can we label the button "Add review"?

Also wondering if we can include feedback once the action is complete? I added a banner in the designs.
![image](https://user-images.githubusercontent.com/41907215/82207502-d70a1280-9901-11ea-8cbf-535f68347286.png)

https://projects.invisionapp.com/share/B9WVPQAS4PE#/screens/417703802_EC-Review-Posted

**Comment by hdrury1** [comment]
Happy for text to be "Add review" but the banner will be work for another ticket pending feedback.
